github-electron/github-electron.d.ts

```diff
@@ -25,6 +25,25 @@ declare namespace Electron {
 		sender: EventEmitter;
 	}
 
+	type integer = number;
```

Comment:
I know there is no integer type in JavaScript, but the actual native code bindings in Electron enforce the integer type.